shared/components/Animation/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@tds/shared-animation",
+  "version": "1.0.0",
```

Comment:
I'm not sure if this should start at `1.0.0` or at `0.1.0`..... Will lerna need to do a release and bump it to version 1?